docs/management/connectors/action-types/gen-ai.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[gen-ai-action-type]]
+== Generative AI connector and action
+++++
+<titleabbrev>Generative AI</titleabbrev>
+++++
+
+The Generative AI connector uses https://github.com/axios/axios[axios] to send a POST request to an OpenAI provider, either OpenAI or Azure OpenAI. The connector uses the <<execute-connector-api,run connector API>> to send the request.
+
+[float]
+[[define-gen-ai-ui]]
+=== Create connectors in {kib}
+
+You can create connectors in *{stack-manage-app} > {connectors-ui}*  For example:
+
+[role="screenshot"]
+image::management/connectors/images/gen-ai-connector.png[Generative AI connector]
+
+[float]
+[[gen-ai-connector-configuration]]
+==== Connector configuration
+
+Generative AI connectors have the following configuration properties:
+
+Name::      The name of the connector.
+API Provider::   The OpenAI API Provider, either OpenAI or Azure OpenAI.
+API URL::   The OpenAI request URL.
+API Key::   The OpenAI or Azure OpenAI API key for authentication.
+
+[float]
+[[preconfigured-gen-ai-configuration]]
+=== Create preconfigured connectors
+
+If you are running {kib} on-prem, you can define connectors by
+adding `xpack.actions.preconfigured` settings to your `kibana.yml` file.
+For example:
+
+[source,text]
+--
+xpack.actions.preconfigured:
+  my-gen-ai:
+    name: preconfigured-gen-ai-connector-type
+    actionTypeId: .gen-ai
+    config:
+      apiUrl: https://api.openai.com/v1/chat/completions
+      apiProvider: 'Azure OpenAI'
+    secrets:
+      apiKey: passwordkeystorevalue
+--
+
+Config defines information for the connector type.
+
+`apiProvider`:: A string that corresponds to *OpenAI API Provider*.
+`apiUrl`:: A URL string that corresponds to the *OpenAI API URL*.
+
+Secrets defines sensitive information for the connector type.
+
+`apiKey`:: A string that corresponds to *OpenAI API Key*.
+
+[float]
+[[gen-ai-action-configuration]]
+=== Test connectors
+
+You can test connectors with the <<execute-connector-api,run connector API>> or
+as you're creating or editing the connector in {kib}. For example:
+
+[role="screenshot"]
+image::management/connectors/images/gen-ai-params-test.png[Generative AI params test]
+
+The Generative AI connector supports one type of action: run (aka test)
```

Comment:
> The Generative AI connector supports one type of action: run (aka test)
> Generative AI run/test ..
> Body:: ...

IMO these three phrases can be replaced by something simple like this:

> You must supply a JSON payload, which is sent to the OpenAI API URL. For example:

Reply:
I'm following the format of other connectors here. `body` is an argument. I will update to be more clear:

```
The Generative AI actions have the following configuration properties.

Body::      A JSON payload sent to the OpenAI API URL. For example: 
```